sandbox/lua/modules/line_protocol.lua

```diff
@@ -0,0 +1,246 @@
+-- This Source Code Form is subject to the terms of the Mozilla Public
```

Comment:
I would also rename this module 'line_protocol' is too generic for and influxDB specific module.